streampark-console/streampark-console-service/src/main/resources/mapper/system/RoleMapper.xml

```diff
@@ -32,8 +32,7 @@
         select r.*
         from t_role r
         left join t_member m on r.role_id = m.role_id
-        left join t_user u on u.user_id = m.user_id
-        where u.user_id = #{userId}
+        where m.user_id = #{userId}
```

Comment:
query condition `team_id` is missing, please check.

Reply:
After check, I don't know why need to call `findUserRole`?

There are two places in the code where roles are used, but I don't know why?

1. `ShiroRealm.doGetAuthorizationInfo` -> `simpleAuthorizationInfo.setRoles(roleSet)`
2. `UserServiceImpl.generateFrontendUserInfo` -> `userInfo.put("roles", roles);` 

Do you know why need roles here? As I understand, front-end just needs the `permissions` and `routers`.

Reply:
> After check, I don't know why need to call `findUserRole`?
> 
> There are two places in the code where roles are used, but I don't know why?
> 
> 1. `ShiroRealm.doGetAuthorizationInfo` -> `simpleAuthorizationInfo.setRoles(roleSet)`
> 2. `UserServiceImpl.generateFrontendUserInfo` -> `userInfo.put("roles", roles);`
> 
> Do you know why need roles here? As I understand, front-end just needs the `permissions` and `routers`.

hi 1996fanrui:

Currently, the `roles` is not actually used,  The front-end has a role-related instruction: `hasRole`, which is used for permission control, but it is not currently used.  Shouldn't use it in the future, so, The method and called references can be deleted.

Reply:
@wolfboys  Thanks for your clarification, updated.

Reply:
> @wolfboys Thanks for your clarification, updated.

👌I'll review it later